skimage/transform/seam_carving.py

```diff
@@ -47,18 +47,18 @@ def seam_carve(img, energy_map, mode, num, border=1, force_copy=True):
            http://www.cs.jhu.edu/~misha/ReadingSeminar/Papers/Avidan07.pdf
     """
 
-    utils.assert_nD(img, (2, 3))
-    image = util.img_as_float(img, force_copy)
+    utils.assert_nD(image, (2, 3))
+    img = util.img_as_float(image, force_copy)
```

Comment:
@jni I don't think that it's worthwhile to do this. We have `rescale_to_*` on its way (https://github.com/scikit-image/scikit-image/issues/1234).